go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/golang/snappy v0.0.5-0.20220116011046-fa5810519dcb
 	github.com/gorilla/websocket v1.5.0
 	github.com/hashicorp/golang-lru/v2 v2.0.7
-	github.com/ipfs/boxo v0.19.0
+	github.com/ipfs/boxo v0.17.0
```

Comment:
Best to use the same version as in the replace statement, `89bceff34bf1` (replace statement refers to https://github.com/onflow/boxo/pull/1, i.e. commit https://github.com/onflow/boxo/commit/f2477b92f4831221274c63960b5d0a65d84180bf; see parent)